packages/backend/prisma/schema.prisma

```diff
@@ -163,6 +163,18 @@ model RefreshToken {
   expiresAt DateTime @db.Timestamp
 }
 
+model VerificationCode {
```

Comment:
Why do you use Postgres for sms/email verification codes. We agreed that Redis should be used. Delete this table, delete mirgation. Use redisServise with expiration time 15 minutes for sms and 24 hours for email.

Reply:
Save verification code algorithm
1. generate verififcation code (you already implemented it)
2. generate random string - key for redis
3. Save verification code and user id to redis `key: randomstring, value: {userId, verificationCode}`
4. Send sms to user.
5. Return random string to frontend (you can use cookies or just body)
6. When user entered code on FE -> send it with that random string to BE
7. Check code is the same, check userId is the same